packages/api/src/routers/auth/mobile/sign_up.ts

```diff
@@ -45,24 +46,24 @@ export async function createMobileSignUpResponse(
 }
 
 export async function createMobileEmailSignUpResponse(
-  email?: string,
-  password?: string,
-  username?: string,
-  name?: string
+  requestBody: any
 ): Promise<JsonResponsePayload> {
   try {
-    if (!email || !password || !username || !name) {
+    if (!isValidSignupRequest(requestBody)) {
```

Comment:
@jacksonh I added the validation here to check the length of the user email address before trimming